### PR TITLE
{lang}[GCCcore/6.4.0] NASM v2.13.01

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (vanzod) [inline easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb]: Replace old URL with `https://github.com/easybuilders`

File: easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+name = 'NASM'
+version = '2.13.01'
+
+homepage = 'http://www.nasm.us/'
+
+description = """NASM: General-purpose x86 assembler"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
```
Comment (vanzod) [inline easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb]: The PIC option is not present in any previous easyconfig file. Is there a specific reason why is it now needed?

File: easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+source_urls = ['http://www.nasm.us/pub/nasm/releasebuilds/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/n/NASM/NASM-2.13.01-GCCcore-6.4.0.eb]: Please add SHA256 checksum

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/b4e9eeb40dfd6d89ea79529159cde439 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @JackPerdue!

**Comment by boegel** [comment]
W.r.t. the use of `pic`: it may be needed if something is linking to libraries provided by this.

I think included it has a small performance/security impact (one negative, one positive I think, I forgot which one is which).

OK as got merged now, but this may pop up again later...

**Pull request merged**